openlibrary/components/BulkSearch/components/BulkSearchControls.vue

```diff
@@ -74,7 +75,11 @@ export default {
         matchBooksText(){
             if (this.loadingMatchedBooks) return 'Loading...'
             return 'Match Books'
-        }
+        },
+        showColumnHint(){
+            if (this.bulkSearchState.activeExtractor) return this.bulkSearchState.activeExtractor.name === 'table_extractor'
+            return false
```

Comment:
This should always work :+1: 
```suggestion
            return this.bulkSearchState.activeExtractor.name === 'table_extractor';
```

Reply:
Resolved